### ISSUE TITLE
AzureHttpRouter does not implement 'getRequestHostname'; breaks `nest build`

### ISSUE THREAD

**Issue by DenzoNL** [nestjs/azure-func-http#119]
## I'm submitting a...

<pre><code>
[ ] Regression 
[X] Bug report
[ ] Feature request
[ ] Documentation issue or request
[ ] Support request => Please do not submit support request here, instead post your question on Stack Overflow.
</code></pre>

## Current behavior

The `@nestjs\azure-func-http\dist\router\azure-http.router.d.ts` type definition is missing the `getRequestHostname`  function declared in `AbstractHttpAdapter`. It is however, present in `lib/router/azure-http.router.ts`. If I manually add it, it all builds and runs just fine.

 I'm thinking a build/publish might have gone wrong?

## Environment

<pre><code>
Nest version: 6.10.14

 
For Tooling issues:
- Node version: v12.14.1 
- Platform: Windows  

Others:

- @nestjs/azure-func-http version: 0.4.1
 (also present in 0.4.0, 0.3.2, haven't tested further back as the property didn't exist back then in AbstractHttpAdapter)
</code></pre>

**Comment by kamilmysliwiec** [comment]
Are you sure that you're on a 0.4.1 version? It should be fixed as a part of this release.

**Comment by rajeevbbqq** [comment]
@kamilmysliwiec Same error. and I'm using `0.4.1`.

![image](https://user-images.githubusercontent.com/16451615/73611906-045dd980-460d-11ea-8f4a-235e81ee551a.png)

**Comment by DenzoNL** [comment]
> Are you sure that you're on a 0.4.1 version? It should be fixed as a part of this release.

My package.lock reads this, so yeah:

```json
    "@nestjs/azure-func-http": {
      "version": "0.4.1",
      "resolved": "https://registry.npmjs.org/@nestjs/azure-func-http/-/azure-func-http-0.4.1.tgz",
      "integrity": "sha512-5Ee0a2oKQDqJq1IVMCxFO33N3Z5s+Z9W2lEY8UpauLsBnKumObxQBFnn+WUXUr2WQIm7AAV6u2r4G+x4C08vcg==",
      "requires": {
        "@angular-devkit/schematics": "8.3.22",
        "@schematics/angular": "8.3.22",
        "cors": "2.8.5",
        "trouter": "3.1.0"
      },
```

I will set up a repro tomorrow at work to make sure.

**Comment by kamilmysliwiec** [comment]
Fixed in 0.4.2
